src/connectors/infinite-hits/connectInfiniteHits.ts

```diff
@@ -91,27 +131,27 @@ export default function connectInfiniteHits(renderFn, unmountFn) {
         })
         .search();
     };
-    const getShowMore = helper => () => {
+    const getShowMore = (helper): (() => void) => () => {
       helper.setPage(lastReceivedPage + 1).search();
     };
 
     return {
       getConfiguration() {
-        return escapeHTML ? TAG_PLACEHOLDER : undefined;
+        return escapeHTML ? TAG_PLACEHOLDER : {};
       },
 
       init({ instantSearchInstance, helper }) {
-        this.showPrevious = getShowPrevious(helper);
-        this.showMore = getShowMore(helper);
-        firstReceivedPage = helper.state.page;
-        lastReceivedPage = helper.state.page;
+        (this as any).showPrevious = getShowPrevious(helper);
```

Comment:
Is there a better way to do this ?

Reply:
We don't need to store it on `this`. We can declare it in an upper-level in a variable.